test/strings/basic.jl

```diff
@@ -478,6 +478,7 @@ Base.endof(x::CharStr) = endof(x.chars)
 @test cmp(CharStr("\U1f596"), "\U1f596\U1f596") == -1
 
 # repeat function
+@inferred repeat(GenericString("x"),1)
```

Comment:
I  think it's fine since the following tests don't space-separate this and the tests files aren't so strict regarding this...

I am wondering if we could bake in auto formatting  using CSTParser at some point